filter_chains.awk

```diff
@@ -1,4 +1,4 @@
+tolower($3 " " $6) ~ /(^|[^a-z])(a.f. blakemore|accor|apple green|ask italian|atalian servest|atlas hotels|bakers . baristas|bannatyne health club|bar . grills|baxterstorey|benugo|bermondsey pub|black sheep coffee|blue diamond|boparan restaurant group|bourne leisure|brains|brewdog|brunning and price|burger king|buzz bingo|ch&co|caff?[èe] nero|cafe nero|center parcs|coffee#1|company of cooks|compass group|costa coffee|crussh|côte brasserie|d&d london|david lloyd clubs|deep blue fish & chips|dobbies garden centres|dunelm|eu garages|east ayrshire council school canteens|elior|english heritage|everyman|everyone active|five guys|franco manca|freedom leisure|fridays|fullers|giggling squid|gourmet burger kitchen|greene king|grosvenor casinos|hall & woodhouse|hollywood bowl|honest burgers|hydes|iss mediclean ltd|jw lees|joe & the juice|joseph holt|jurys inn|kfc|kew green hotels|krispy kreme doughnuts|laine|leon restaurants|loungers|m&s cafe|macdonald hotels|marston['’]?s|mcdonalds|mcmullen|mecca|miller & carter|mitchells & butlers|morrisons - cafe and barista bar|moto|nando[’']?s|new world trading company|olive dining|park holidays|parkdean resorts|patisserie valerie|pho|pizza express|pizza hut|powerleague|pret a manger|prezzo|revolution bars|roadchef|ssp group|selfridges|shepherd neame|st austell brewery|starbucks|stonegate|team sport|tenpin|the big table|the city pub group|the cornish bakery|the haven|the ivy collection|the national trust|the restaurant group|tony macaroni|tortilla|turtle bay|vacherin|village hotels|viva italia|wildwood|wadworth|wagamama|wasabi|welcome break|wetherspoons?|whitbread|wilson vale|yo sushi|youngs|zizzi)($|[^a-z])/ {
```

Comment:
CLARIFICATION: My only worry would be about false positives:  All these would be included:

```
$ grep "ST15 0EU" ~/dev/static-distance/target/complete.csv | cut -f3,6,7,9 -d,
BP Express Shopping Ltd,BP Stafford SFC  M6 Motorway Service Area,Northbound,ST15 0EU
Burger King,MOTO Motorway Service Area,M6 North,ST15 0EU
Chow,Moto Motorway Service Area,M6 North,ST15 0EU
Costa Coffee,MOTO Motorway Service Area,M6 North,ST15 0EU
Costa Drive Through,MOTO Motorway Services Area,M6 North,ST15 0EU
Greggs,MOTO Motorway Service Area,M6 North,ST15 0EU
Marks & Spencer Simply Food,MOTO Motorway Service Area,M6 North,ST15 0EU
Stafford Travelodge,MOTO Motorway Service Area,M6 North,ST15 0EU
West Cornwall Pasty,MOTO Motorway Service Area,M6 North ,ST15 0EU
WH Smith,MOTO Motorway Service Area,M6 North,ST15 0EU
```
(because it's MOTO in address 1) - I'm not sure whether that's correct - "West Cornwall Pasty" or "WH Smith"